rust/linux-client/src/main.rs

```diff
@@ -97,7 +97,7 @@ fn system_resolvers(dns_control_method: Option<DnsControlMethod>) -> Result<Vec<
 
 #[derive(Clone)]
 struct CallbackHandler {
-    handle: Option<file_logger::Handle>,
+    _handle: Option<file_logger::Handle>,
```

Comment:
We should be able to remove this field and store the handle on the stack of `main` to not have it go out of scope.

Reply:
👍